enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -72,21 +78,41 @@
         (log/error e (trs "ERROR LOAD from {0}: {1}" path (.getMessage e)))
         (throw e)))))
 
-(mu/defn v2-load
+(mu/defn ^:private v2-load-internal
```

Comment:
Small suggestion, maybe this could just be the entry point for internal use cases, rather than having a separate `v2-load-no-token-check` wrapper? And then the token check could just be done unconditionally in `v2-load`